compiler-cli/src/run.rs

```diff
@@ -73,6 +122,9 @@ fn run_erlang(package: &str, module: &str, arguments: Vec<String>) -> Result<i32
         args.push(entry.path().join("ebin").to_string_lossy().into());
     }
 
+    // gleam modules are seperated by `/`. Erlang modules are seperated by `@`.
+    let module = module.replace('/', "@");
```

Comment:
We also need to update the JavaScript running functions. If the module is not in the same package then we'll need to have a different import path to support this.

Reply:
This was fixed my modifying the `write_javascript_entrpoint` function to use the module name that the main function was instead of the module from config.